src/AstromFit.cc

```diff
@@ -502,111 +502,111 @@ and I did not want to replicate it.  The constness of the iterators is
+                               mjd);
+
+        Eigen::Vector2d res(fittedStarInTP.x - outPos.x, fittedStarInTP.y - outPos.y);
+        double chi2Val = res.transpose() * transW * res;
+
+        accu.AddEntry(chi2Val, 2, &ms);
     }// end of loop on measurements
 }
 
 
 //! for a list of images.
 template <class ListType, class Accum>
-void AstromFit::AccumulateStatImageList(ListType &L, Accum &Accu) const
+void AstromFit::AccumulateStatImageList(ListType &list, Accum &accum) const
 {
-  for (auto im=L.begin(); im!=L.end() ; ++im)
+    for (auto im = list.begin(); im != list.end() ; ++im)
     {
-      AccumulateStatImage(**im, Accu);
+        AccumulateStatImage(**im, accum);
     }
 }
 
 template <class Accum>
-void AstromFit::AccumulateStatRefStars(Accum &Accu) const
+void AstromFit::AccumulateStatRefStars(Accum &accum) const
 {
-  /* If you wonder why we project here, read comments in
-     AstromFit::LSDerivatives2(TripletList &TList, Eigen::VectorXd &Rhs) */
-  FittedStarList &fsl = _assoc.fittedStarList;
-  TanRaDec2Pix proj(GtransfoLin(), Point(0.,0.));
-  for (auto i = fsl.begin(); i!= fsl.end(); ++i)
+    /* If you wonder why we project here, read comments in
+       AstromFit::LSDerivatives2(TripletList &TList, Eigen::VectorXd &Rhs) */
+    FittedStarList &fsl = _assoc.fittedStarList;
+    TanRaDec2Pix proj(GtransfoLin(), Point(0., 0.));
+    for (auto i = fsl.begin(); i != fsl.end(); ++i)
     {
-      FittedStar &fs = **i;
-      const RefStar *rs = fs.GetRefStar();
-      if (rs == NULL) continue;
-      proj.SetTangentPoint(fs);
-      // fs projects to (0,0), no need to compute its transform.
-      FatPoint rsProj;
-      proj.TransformPosAndErrors(*rs, rsProj);
-      // TO DO : account for proper motions.
-      double rx = rsProj.x; // -fsProj.x (which is 0)
-      double ry = rsProj.y;
-      double det = rsProj.vx*rsProj.vy - sqr(rsProj.vxy);
-      double wxx = rsProj.vy/det;
-      double wyy = rsProj.vx/det;
-      double wxy = -rsProj.vxy/det;
-      wxx *= HACK_REF_ERRORS;
-      wyy *= HACK_REF_ERRORS;
-      wxy *= HACK_REF_ERRORS;
-      Accu.AddEntry(wxx*sqr(rx) + 2*wxy*rx*ry+ wyy*sqr(ry), 2, &fs);
+        FittedStar &fs = **i;
+        const RefStar *rs = fs.GetRefStar();
+        if (rs == NULL) continue;
+        proj.SetTangentPoint(fs);
+        // fs projects to (0,0), no need to compute its transform.
+        FatPoint rsProj;
+        proj.TransformPosAndErrors(*rs, rsProj);
+        // TO DO : account for proper motions.
+        double rx = rsProj.x; // -fsProj.x (which is 0)
+        double ry = rsProj.y;
+        double det = rsProj.vx * rsProj.vy - sqr(rsProj.vxy);
+        double wxx = rsProj.vy / det;
+        double wyy = rsProj.vx / det;
+        double wxy = -rsProj.vxy / det;
+        wxx *= HACK_REF_ERRORS;
+        wyy *= HACK_REF_ERRORS;
+        wxy *= HACK_REF_ERRORS;
+        accum.AddEntry(wxx * sqr(rx) + 2 * wxy * rx * ry + wyy * sqr(ry), 2, &fs);
```

Comment:
Consider removing the whitespace around the multiplication operators (but leave the rest!)